Doc/library/dis.rst

```diff
@@ -210,6 +210,13 @@ operation is being performed, so the intermediate analysis object isn't useful:
    This generator function uses the ``co_firstlineno`` and ``co_lnotab``
    attributes of the code object *code* to find the offsets which are starts of
    lines in the source code.  They are generated as ``(offset, lineno)`` pairs.
+   Functions decoding directly ``co_lnotab`` should use a signed
```

Comment:
Too much details for common user of the dis module, but too few details for reproducing decoding algorithm. I think just adding the reference to `Objects/lnotab_notes.txt` is enough.